Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -0,0 +1,97 @@
+1. Start time: First day of your holiday 
+1. End time: Last day of your holiday
+
+### Before leaving
+
+When we leave on holidays we should make sure we leave our work either finished or delegated to someone else. There are a couple of steps that should be performed to reduce the impact of the absence.
+
+1. Try to finish any work in progress before you go.
+1. Resolve the reviews on your opened PRs and try to get them merged.
+1. In the event of not being able to complete 1. and/or 2. please sync with another iOS Engineer, ideally someone from your squad, and do a handover. When handing over an opened PR please make sure you add the iOS Engineer as `Assignee`.
+1. [Add yourself to the PullReminder exclusion list](#steps-to-be-added-to-the-pullreminder-exclusion-list) to prevent you to be added as a PR reviewer while you are away. 
+1. Once you get back, you have to remove yourself from the PullReminder exclusion list. Adding a Slack reminder (`/remind me to reactivate PullReminder on <date>`) before you leave might help you not to forget.
+
+#### Steps to be added to the PullReminder exclusion list
+1. Go to [pullreminders.com](https://pullreminders.com)
+1. Sign in
+1. Select Babylonpartners organization
+1. Select iOS-PullAssigners team
+1. Add yourself to the Excluded team members
+
+## Working from Home 🏡
+This section is only valid for non-remote employees.
+
+Working from home is a benefit and should be seen as such. 
+At the moment it is accepted that an engineer, once settled in the iOS Team processes, project, and squad is allowed to work from home 1-2 days per week. 
+The number of days may very depending on your personal situation and your squad’s way of working. These should be discussed and agreed with your Line Manager and your PM.
+
+When working from home you should notify in the the iOS team Slack channel when you are away and when are you expecting to come back.
+
+### Request
+
+1. Ask for approval from your PM - only if approved you can proceed to the next steps.
+2. Communicate it to your line manager and mention your PM approval
+3. Request in Bamboo
+
+### Notify
+
+1. Notify the iOS Team and your Squad via Slack on the working day before you are working from home. Emergencies might happen that require you to work from home and only notify on the same day. That will be acceptable if there is a reasonable excuse for it.
+2. On the day you are working from home change your Slack status to 🏡 **Working remotely**.
+
+### Being remote
+
+For the majority of the engineers in the team, the office is our usual place of work. We are used to work next to our colleagues and is natural that we end up communicating more verbally. In the office everyone in your Squad is aware of where you are and what you are doing, so there is no need for us to keep notifying each other.
+
+Working remotely requires a different mindset. We have to bear in mind that people in the office should still be aware of what is happening so you should:
```

Comment:
tbh that should be true for non-remote as well, in the sense that non-remote people should be aware that their collegues are remote (some of us even are full-time remote). So that sentence above:

> We are used to work next to our colleagues and is natural that we end up communicating more verbally. In the office everyone in your Squad is aware of where you are and what you are doing, so there is no need for us to keep notifying each other.

is not really true and a bit annoying to read that it could be the mindset that people usually working in the office could have. Because in practice everybody should already be aware that a lot of people are working remote every day (either people used to work in office but WFH, or full-time remote) and even people in the office should communicate clearly via slack, to the remote people.

Reply:
That's true. I will re-phrase that. This was more directed to squads that have just non-remote people.